packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -112,7 +112,7 @@ class MaterialTextSelectionControls extends TextSelectionControls {
   @override
   Offset getHandleAnchor(TextSelectionHandleType type, double textLineHeight) {
     return switch (type) {
-      TextSelectionHandleType.collapsed => const Offset(_kHandleSize / 2, -4),
+      TextSelectionHandleType.collapsed => const Offset(_kHandleSize / 2.18, -4.5),
```

Comment:
Can you add a comment explaining how you came up with these values? Something like:

```
// These values were eyeballed to match a physical Pixel 9 Running Android 16.
```